composite/market-closure/src/price.ts

```diff
@@ -0,0 +1,30 @@
+import { Execute } from '@chainlink/types'
+import finnhub from '@chainlink/finnhub-adapter'
+import fcsapi from '@chainlink/fcsapi-adapter'
+
+export type PriceDataOptions = { type?: PriceDataProvider }
```

Comment:
We don't need this `PriceDataOptions` if it holds just the type. We can use the type directly. This boilerplate is from the Proof Of Reserves composite adapter, and we can also remove it there.
```suggestion
```